<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMultipleTorrents</class>
 <widget class="QDialog" name="AddMultipleTorrents">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add multiple torrents</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/images/bittorrent.svg</normaloff>:/resources/images/bittorrent.svg</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Tags:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="LC::Util::TagsLineEdit" name="TagsEdit_"/>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="SaveDirectory_"/>
       </item>
       <item>
        <widget class="QPushButton" name="BrowseSave_">
         <property name="text">
          <string>Browse...</string>
         </property>
         <property name="icon">
          <iconset theme="document-save-all"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QLineEdit" name="OpenDirectory_"/>
       </item>
       <item>
        <widget class="QPushButton" name="BrowseOpen_">
         <property name="text">
          <string>Browse...</string>
         </property>
         <property name="icon">
          <iconset theme="document-open-folder"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Open directory:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Save directory:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Additional parameters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="AddAsStarted_">
        <property name="text">
         <string>Start torrents after adding</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="OnlyIfExists_">
        <property name="toolTip">
         <string>If this option is checked, the only torrents added are those for which at least some files are already downloaded. Namely, for a single-file torrent it checks the presentce of the corresponding file in the destination path, and for a multi-file torrent it checks the presence of the root directory of the torrent.

This is useful, for example, when you need to re-add a bunch of torrents you were already seeding after the tracker has changed the torrent files, requiring you to re-download them.</string>
        </property>
        <property name="whatsThis">
         <string>If this option is checked, the only torrents added are those for which at least some files are already downloaded. Namely, for a single-file torrent it checks the presentce of the corresponding file in the destination path, and for a multi-file torrent it checks the presence of the root directory of the torrent.

This is useful, for example, when you need to re-add a bunch of torrents you were already seeding after the tracker has changed the torrent files, requiring you to re-download them.</string>
        </property>
        <property name="text">
         <string>Add a torrent only if its files already exist</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC::Util::TagsLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">util/tags/tagslineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddMultipleTorrents</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddMultipleTorrents</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
